packages/components/pages/f-self-exclusion/src/components/_tests/SelfExclusion.test.js

```diff
@@ -0,0 +1,33 @@
+import Vuex from 'vuex';
+import { VueI18n } from '@justeat/f-globalisation';
+import { shallowMount, createLocalVue } from '@vue/test-utils';
+import SelfExclusion from '../SelfExclusion.vue';
+import {
+    createStore,
+    i18n,
+    $log
+} from './helpers/setup';
+
+const localVue = createLocalVue();
+
+localVue.use(VueI18n);
+localVue.use(Vuex);
+
+describe('SelfExclusion', () => {
```

Comment:
Based on what I can see in the template, it seems like there is some conditional rendering that could potentially be tested here as well as some user actions. Are there plans to add tests for these bits?

Reply:
No, we do not intend to add any tests for this component. We have a QA who will test the functionality.

Reply:
This seems dangerous to me, but it's entirely up to you